common/src/test/java/org/opensearch/ml/common/MLCommonsClassLoaderTests.java

```diff
@@ -183,7 +185,7 @@ public void testClassLoader_MLInput() throws IOException {
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testConnectorInitializationException() {
+    public void testConnectorInitializationException() throws JsonParseException {
```

Comment:
Please add the test case which actually throws JsonParseException, the actual scenario which you are fixing this PR.

Reply:
Looks good to me.